config/elasticsearch/mappings/feed_content.json

```diff
@@ -170,9 +170,6 @@
         },
         "profile_image_90": {
           "type": "keyword"
-        },
-        "pro": {
-          "type": "boolean"
```

Comment:
Our ES docs made it sound like it is safe/acceptable to edit these mapping files directly, so I went ahead and removed the `pro` key from them. Let me know if I misunderstood and actually shouldn't be touching these files for some reason!

Reply:
No reindexing needed. The field will still exist in the index and on the documents until they are reindexed naturally but it won't hurt anything. If we had billions of documents and were keeping ES then it would be worth it to explicitly remove it to save space but in our case, no additional action is needed here.